conans/client/conf/__init__.py

```diff
@@ -230,19 +231,11 @@ def storage(self):
 
     @property
     def storage_path(self):
-        try:
-            conan_user_home = os.getenv("CONAN_USER_HOME")
-            if conan_user_home:
-                storage = self.storage["path"]
-                if storage[:2] == "~/":
-                    storage = storage[2:]
-                result = os.path.join(conan_user_home, storage)
-            else:
-                result = conan_expand_user(self.storage["path"])
-        except KeyError:
-            result = None
-        result = get_env('CONAN_STORAGE_PATH', result)
-        return result
+        store_path = os.path.expanduser(self.storage.get("path", "data"))
```

Comment:
I don't get the logic of this method. ``storage.get("path", "data")``?
Also, what happened to ``CONAN_STORAGE_PATH``?